src/Uno.UI/Extensions/ViewExtensions.visual-tree.cs

```diff
@@ -199,13 +199,13 @@ internal static IEnumerable<_View> EnumerateAncestors(this _View? o)
 		}
 	}
 
-	private static IEnumerable<_View> EnumerateChildren(this _View? o)
+	internal static IEnumerable<_View> EnumerateChildren(this _View? o)
```

Comment:
👍 Considering the namespace, I would even consider to add a public version that is not an extension method ... we need those kind of methods also in runtime-tests for instance ^^